JavaScript/JavaScriptSDK.Tests/CheckinTests/Telemetry/PageViewPerformance.tests.ts

```diff
@@ -17,6 +17,7 @@ class PageViewPerformanceTelemetryTests extends ContractTestHelper {
             test: () => {
 
                 var telemetry = new Microsoft.ApplicationInsights.Telemetry.PageViewPerformance("name", "url", 0);
+                Assert.equal(true, telemetry.isValid);
```

Comment:
Can we create a separate test that verifies default value of `isValid` instead of adding assertions to multiple existing tests?